plugins/techdocs/src/reader/transformers/sanitizeDOM.test.ts

```diff
@@ -111,6 +112,108 @@ describe('sanitizeDOM', () => {
     expect(shadowDom.querySelectorAll('link').length).toEqual(1);
   });
 
+  it('render iframe where src host is in allowedIframeHosts', async () => {
+    const html = `
+      <html>
+        <head>
+          <link rel="stylesheet" href="style.css">
+        </head>
+        <body>
+          <iframe src="https://example.com?test=1"></iframe>
+          <iframe src="https://forbidden.com?test=1"></iframe>
+        </body>
+      </html>
+    `;
+    const config = new ConfigReader({
+      allowedIframeHosts: ['example.com'],
+    });
+    const shadowDom = await createTestShadowDom(html, {
+      preTransformers: [sanitizeDOM(config)],
+      postTransformers: [],
+    });
+    expect(shadowDom.querySelectorAll('link').length).toEqual(1);
+    expect(shadowDom.querySelectorAll('iframe').length).toEqual(1);
```

Comment:
can we add an assertion here to make sure the src is equal to `https://example.com?test=1` as well?

Reply:
@emmaindal done :)

Reply:
⭐